fearless/Common/Configs/ApplicationConfigs.swift

```diff
@@ -107,4 +107,8 @@ extension ApplicationConfig: ApplicationConfigProtocol {
     var learnControllerAccountURL: URL {
         URL(string: "https://wiki.polkadot.network/docs/en/maintain-guides-how-to-nominate-polkadot#setting-up-stash-and-controller-keys")!
     }
+
+    var subscanAPIHeaderKey: String {
+        "X-API-Key"
```

Comment:
Just put this guy straight to the Request header. Motivation: this is not the application config but part of the protocol to communicate with subscan and should reside inside implementation. You could have a constant in the SubscanOperation factory if you wish :)